public/javascripts/create-password.js

```diff
@@ -0,0 +1,75 @@
+const continueEvent = document.querySelector("#continue-button");
+const passwordFields = document.querySelector("#password-fields");
+const formGroup = document.querySelector(".govuk-form-group");
+const passwordInput = document.querySelector("#password");
+const confirmPasswordInput = document.querySelector("#confirm-password");
+const errorTextSection = document.querySelector("#error");
+const errorSummarySection = document.querySelector("#error-summary");
+
+continueEvent.addEventListener("click", checkPassword);
```

Comment:
nit: I think this should be at the end of the document because the function it uses hasn't been created yet (#readability)